src/Summarizer.js

```diff
@@ -43,6 +43,22 @@ class Summarizer{
 		return result_string;
 	}
 
+	summaryToArray(sorted_sentences, clean_sentences){
```

Comment:
Could you write a jsdoc for this to help the documentation effort?

Reply:
These methods aren't directly accessed by the user, rather they are called by the `summarizeByRank`and `summarizeByFrequency` methods. What do you think I should really write for them?

Reply:
I was thinking for future developers so it's easier to see and understand the code, but a comment is perfect! Thank you :)